### PR TITLE
Bug 1928869: remove css overrides for search component and update pf dependency to 2021.2

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1928869, https://bugzilla.redhat.com/show_bug.cgi?id=1929281

Thanks to @dtaylor113 for the tech preview badge fix

### CODE REVIEW

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by jcaianirh** [comment]
/hold remove 
officially released: https://staging.patternfly.org/v4/developer-resources/release-notes#2021.02-release-notes-2021-02-17

**Comment by jcaianirh** [comment]
/hold cancel
officially released: https://staging.patternfly.org/v4/developer-resources/release-notes#2021.02-release-notes-2021-02-17

File: frontend/packages/console-shared/src/components/badges/Badge.scss [hunk 1]
```diff
@@ -9,6 +9,9 @@
   &.pf-c-label {
     background-color: #d93f00;
     border-radius: var(--pf-global--BorderRadius--sm);
+    :before {
+      border-color: transparent;
```
Comment (spadgett) [inline frontend/packages/console-shared/src/components/badges/Badge.scss:13]: Hey @jcaianirh, can you comment on why this is needed?

Comment (jcaianirh) [reply frontend/packages/console-shared/src/components/badges/Badge.scss:13]: @spadgett sure, The pf label css changed slightly to work better on a gray background: https://github.com/patternfly/patternfly/pull/3817. But we customize the css for the label in the tech preview badge css so it conflicts with the pf css change. The tech preview badge is squared off, and the pf has a capsule shape, so you can see a capsule border without this change (@dtaylor113 caught this while testing pf-2021.2 with openshift):
image (2)

Comment (spadgett) [reply frontend/packages/console-shared/src/components/badges/Badge.scss:13]: Thanks. I don't know the history here, but I wonder if we shouldn't be using the PF label style as-is.

cc @christianvogt @andrewballantyne @serenamarie125

**Review by spadgett — APPROVED** [review]
/lgtm

Thanks @jcaianirh. Let's go with this change. If we decide to adjust any label styles, we can do that separately.

File: frontend/packages/console-shared/src/components/badges/Badge.scss (hunk 1 above)
Comment (andrewballantyne) [reply frontend/packages/console-shared/src/components/badges/Badge.scss:13]: Not sure why we used a `Label` component originally 🤔 ... predates me.

I'll log a tech debt item to swing back and try to clean this up. It's odd usage of the `Label` component then to just override its styles.

Thanks for the heads up!

**Pull request merged**